pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -308,6 +309,11 @@ let
       # To fix the build as we don't provide libffi_pic.a
       # (ld.lld: error: unable to find library -l:libffi_pic.a):
       use_system_libffi = true;
+    } // optionalAttrs (chromiumVersionAtLeast "115") {
+      # Use nixpkgs Rust compiler instead of the one shipped by Chromium.
+      # We do intentionally not set rustc_version as nixpkgs will never do incremental
+      # rebuilds, thus leaving this empty is fine.
+      rust_sysroot_absolute = "${rustc}";
```

Comment:
```suggestion
      rust_sysroot_absolute = rustc;
```

Reply:
This won't as `lib.isString` is not true for rustc (as it's a derivation) and thus mkGnFlags doesn't like it. See also the LLVM path above.

Reply:
Shouldn't we then use toString?